docs/templates/configuration.md.yml

```diff
@@ -75,6 +75,22 @@ configOptions:
           // This key will be extracted
           myT("key0");
           ```
+  - name: transComponentNames
+    type: "{sourceModule: string; importName: string}[]"
+    description: Possible import names of `Trans` components. Note that, for now, it does a pretty dumb job at resolving the source module so you should list all ways you have to import your components.
```

Comment:
I think this is too much of a limitation, even for a MVP. In particular, having `import {CustomTrans} from "./customTransComponent"` work when `import {CustomTrans} from "../customTransComponent"` doesn't would be very confusing in my opinion. It should resolve relative modules and I'd say it should be relative to the `.babelrc` or the `package.json`.

Reply:
OK, I've made it work for relative imports. However:
- I'm not sure to which file it's related (I looked at outputPath's doc that does not say much either)
- it won't work if callers use weird ways of getting their modules (e.g. `../components/i18n` while being in the components folder) as it's only trying the most direct relative path. However going further would require to rewrite the referencesImport function